backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -167,7 +169,7 @@ public Boolean setupNativeIntegration(BacktraceBase client, BacktraceCredentials
     public Boolean setupNativeIntegration(BacktraceBase client, BacktraceCredentials credentials,
                                           boolean enableClientSideUnwinding, UnwindingMode unwindingMode) {
         // avoid initialization when database doesn't exist
-        if (getSettings() == null) {
+        if (_enable == false || getSettings() == null) {
```

Comment:
`if (!_enable || getSettings() == null) {`

Reply:
I will stick to the current implementation